linux_os/guide/system/software/updating/ensure_gpgcheck_never_disabled/rule.yml

```diff
@@ -20,6 +20,7 @@ identifiers:
 references:
     stigid@rhel6: RHEL-06-000015
     srg@rhel6: SRG-OS-000103
+    srg@rhel7: SRG-OS-000366-GPOS-00153
```

Comment:
@cipherboy this can be just `srg`.

Reply:
no, it can't, then it's srg also for all other OSes

Reply:
which is valid that it is the SRG for other OSes.